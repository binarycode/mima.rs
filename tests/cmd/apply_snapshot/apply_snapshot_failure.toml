args = 'apply-snapshot zero root'
status.code = 1
