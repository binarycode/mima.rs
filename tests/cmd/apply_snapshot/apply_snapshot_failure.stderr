error: Failed to run 'ssh -o BatchMode=yes -o ConnectTimeout=10 -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@10.210.0.13 qemu-img snapshot -aroot /mima/zero-sda.qcow2'

stdout:
qemu-img: Could not open /mima/zero-sda.qcow2: Could not open '/mima/zero-sda.qcow2': No such file or directory

